lib/server.js

```diff
@@ -164,8 +172,7 @@ server.onPhantomPageCreate = function(req, res) {
     this.clearLocalStorage(req, res);
     this.clearMemoryCache(req, res);
 
-    req.prerender.page.run(_this.options.blockedResources || blockedResources, !!_this.options.logRequests, USER_AGENT, function(blockedResources, logRequests, ua, resolve, reject) {
-
+    req.prerender.page.run(args, function(options, resolve, reject) {
```

Comment:
@panthony within this anonymous function, variables defined out of the scope are *not* available. The args and the anonymous function are serialized using json and sent to to Phantom JS, which in turns deserializes it run the serialized function with the serialized args.